### PR TITLE
Update ptd.net config

### LINKED ISSUE #40: Add outgoing servers and update documentation URL for ptd.net
The ptd.net configuration is missing outgoing servers for port 465/SSL and port 25/STARTTLS. Because of changes to the www.ptd.net website, the documentation URL should be updated.

### PR DESCRIPTION
**Pull request by serock**
Fixes #40 
- Added outgoingServer with port 465 / SSL.
- Updated the documentation URL and description.
- Corrected indentation.

### CODE REVIEW

**Comment by serock** [comment]
@benbucksch - When you have time, would you please review this pull request and let me know if there is anything I need to do. Thank you.

File: ispdb/ptd.net [hunk 1]
```diff
@@ -1,46 +1,60 @@
...
+        <incomingServer type="pop3">
+            <hostname>promail.ptd.net</hostname>
+            <port>110</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>465</port>
```
Comment (cketti) [inline ispdb/ptd.net:37]: The linked documentation doesn't list this port as being supported for SMTP. It did work in my test. Still, I wouldn't include it unless it is documented by the provider.

File: ispdb/ptd.net [hunk 2]
```diff
@@ -1,46 +1,60 @@
...
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>587</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </outgoingServer>
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>25</port>
```
Comment (cketti) [inline ispdb/ptd.net:51]: An email client should never have to use port 25 for submission. They do list this in their documentation. But I wouldn't include it here.

Comment (serock) [reply ispdb/ptd.net]: @cketti - Thank you for the feedback. I pushed a commit that removes port 25.

**Comment by serock** [comment]
@benbucksch - Thank you for the feedback. I pushed a commit that removes port 25.

File: ispdb/ptd.net (hunk 1 above)
Comment (benbucksch) [reply ispdb/ptd.net:37]: @cketti Our policy is: prefer normal (implicit) TLS over STARTTLS whenever possible. It's faster connected. Our policy also is to list all available and working configs, not just those in the official docs.

**Review by benbucksch — APPROVED** [review]
Thanks!

**Pull request merged**